### ISSUE TITLE
Dupe files on NZBGet

### ISSUE THREAD

**Issue by pedrolara-boop** [Tensai75/NZBDonkey#6]
NZBGet ignores the DupeCheck for files that I send via NZBDonkey.
Is there a way to avoid this?

I have the DupeCheck turned ON on NZBGet, and it works for files that I drop on the watched folder or manually load.

**Comment by pedrolara-boop** [comment]
The files sent are check to force the duplicate mode. 
Is there a way to change this to the default score?

**Comment by Tensai75** [comment]
Yes, NZBDonkey does ignore the dupe check. It was my understanding that someone actively clicking a NZBlnk actually does want this file to be downloaded.
However, I will keep this in mind as a possible new settings option for NZBGet in a further release.

**Comment by Tensai75** [comment]
Addressed in [v1.0.0](https://github.com/Tensai75/NZBDonkey/releases/tag/v1.0.0) (addition of dupe mode selection in NZBGet configuration)
